contracts/security/src/storage/asset.rs

```diff
@@ -0,0 +1,1041 @@
+use cosmwasm_std::{Addr, Empty, Storage, Uint64};
+use cw_storage_plus::{Bound, Map};
+
+use crate::core::{
+    constants::{
+        ASSET_SECURITY_KEY, DEFAULT_WITH_SECURITY_LIMIT, MAX_WITH_SECURITY_LIMIT,
+        SECURITY_TO_ASSET_KEY,
+    },
+    error::ContractError,
+    msg::{CategorizedSecurity, Paginate, Security},
+};
+
+pub const ASSET_TO_SECURITY: Map<&Addr, Security> = Map::new(ASSET_SECURITY_KEY);
+pub const SECURITY_TO_ASSET: Map<(&str, &str, &Addr), Empty> = Map::new(SECURITY_TO_ASSET_KEY);
+
+/// Attempts to get the asset's security that is stored within the contract's storage.
+///
+/// # Arguments
+///
+/// * `storage` - A reference to the Deps' Storage object.
+/// * `asset_addr` - The address of the asset to retrieve the tag of.
+///
+/// # Examples
+/// ```
+/// let addr = Addr::unchecked("addr1");
+/// get_security(deps.storage, &addr)?;
+/// `
```

Comment:
This doc annotation needs a triple tilde instead of a single to close code block. The other functions in this file need the same fix.